packages/api/src/graphql/mutation.private.ts

```diff
@@ -725,10 +725,16 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       async resolve(root, {id}, {dbAdapter, loaders}) {
         const article = await loaders.articles.load(id)
 
-        if(!article) throw new NotFound('article', id)
+        if (!article) throw new NotFound('article', id)
```

Comment:
Did you do any functional changes in this file? If not please don't add it to the PR.

Reply:
Tried solving this problem but to no avail. Actually I figured that it might not even be a problem in that sense because if you look in this file at `publishPage` for example there's also a space between the `if` and the condition. So prettier actually only does what it's supposed to do (in my understanding at least). But if you'd like I can try to edit this manually from within Github?